packages/scaffolding-cli/templates/build/azDevops/azure/azure-pipelines-ssr-gke.yml

```diff
@@ -313,7 +302,7 @@ stages:
                         parameters:
                           env_vars:
                             APP_BASE_URL: "https://app.$(base_domain)"
-                            MENU_API_URL: "http://dev.amidostacks.com/api/menu"
+                            MENU_API_URL: "https://api.demo.nonprod.amidostacks.com/api/menu"
                             APP_BASE_PATH: "/web/stacks"
                             NODE_ENV: production
                           workingDirectory: $(Agent.BuildDirectory)/s/$(self_repo)/test/testcafe
```

Comment:
on success can be removed. Redundant. There is a independent `dependsOn` pipeline now.

Reply:
are you sure? completely removed?